File: benchmarks/fair/fib.rs
fn fib(n:i64)->i64{ if n<=1 {n} else {fib(n-1)+fib(n-2)} }
fn main(){
    let n:i64=std::env::var("BENCH_N").ok().and_then(|v|v.parse().ok()).filter(|&v|v>0).unwrap_or(32);
    println!("{}",fib(n));
}

File: benchmarks/fair/fib.cs
using System;
class fib {
    static long F(long n) { return n <= 1 ? n : F(n - 1) + F(n - 2); }
    static void Main() {
        long n = 32;
        string e = Environment.GetEnvironmentVariable("BENCH_N");
        if (e != null && long.TryParse(e, out long v) && v > 0) n = v;
        Console.WriteLine(F(n));
    }
}
